docs/guides/cluster.mdx

```diff
@@ -22,20 +22,29 @@ This guide shows how to deploy a distributed Restate cluster consisting of 3 nod
 To deploy a 3 node distributed Restate cluster, copy the `docker-compose.yml` and run `docker compose up`.
 
 ```yaml
+# Common environment variables for all restate nodes
```

Comment:
I _think_, and it's just a personal opinion, that the brevity of this example is a feature and not a bug. Perhaps we can link to `/operate/configuration/server` for the full reference rather than commenting it extensively?

Reply:
> Perhaps we can link to /operate/configuration/server for the full reference rather than commenting it extensively?

That's sort of what i was trying to avoid here, having people to go dig in a longer docs page just to understand the few things they most likely need to tweak